examples/data-sources/gitlab_project_membership/data-source.tf

```diff
@@ -0,0 +1,15 @@
+# By project's ID
+data "gitlab_project_membership" "example" {
+  project_id = 123
+}
+
+# By project's full path
+data "gitlab_project_membership" "example" {
+  full_path = "foo/bar"
+}
+
+# Only get members with access level "owner"
+data "gitlab_project_membership" "example" {
+  project_id   = 123
+  access_level = "owner"
```

Comment:
Can you take another look at your examples? `access_level` isn't an attribute anymore.